updates/35/updates.sql

```diff
@@ -639,4 +639,180 @@ INSERT INTO assets(identifier, name, type) VALUES('SBTC', 'Super Bitcoin', 'W');
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:8453/erc20:0x3e31966d4f81C72D2a55310A6365A56A4393E98D', 'WorldMobileToken', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:8453/erc20:0x3e31966d4f81C72D2a55310A6365A56A4393E98D', 'A', 8453, '0x3e31966d4f81C72D2a55310A6365A56A4393E98D', 6, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:8453/erc20:0x3e31966d4f81C72D2a55310A6365A56A4393E98D', 'WMTX', 'world-mobile-token', 'WMTX', NULL, 1713552365, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x8802269D1283cdB2a5a329649E5cB4CdcEE91ab6', 'Fight to MAGA', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0x8802269D1283cdB2a5a329649E5cB4CdcEE91ab6', 'A', 1, '0x8802269D1283cdB2a5a329649E5cB4CdcEE91ab6', 9, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0x8802269D1283cdB2a5a329649E5cB4CdcEE91ab6', 'FIGHT', 'fight-to-maga', '', NULL, 1720918991, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x77777FeDdddFfC19Ff86DB637967013e6C6A116C', 'TornadoCash', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0x77777FeDdddFfC19Ff86DB637967013e6C6A116C', 'A', 1, '0x77777FeDdddFfC19Ff86DB637967013e6C6A116C', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0x77777FeDdddFfC19Ff86DB637967013e6C6A116C', 'TORN', 'tornado-cash', 'TORN', NULL, 1608260437, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:56/erc20:0x1bA8D3C4c219B124d351F603060663BD1bcd9bbF', 'TornadoCash', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:56/erc20:0x1bA8D3C4c219B124d351F603060663BD1bcd9bbF', 'A', 56, '0x1bA8D3C4c219B124d351F603060663BD1bcd9bbF', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:56/erc20:0x1bA8D3C4c219B124d351F603060663BD1bcd9bbF', 'TORN', 'tornado-cash', 'TORN', NULL, 1631949829, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'Everclear', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'A', 1, '0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'CLEAR', 'everclear', '', NULL, 1693777295, NULL);
```

Comment:
seems that we have the old coingecko/cc  Don't we need to update them?
![image](https://github.com/user-attachments/assets/5d6b8540-509b-4dfa-88b2-e949854bc051)

Reply:
i think this is incorrect. 
```sh
sqlite> select * from assets where identifier='eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8';
sqlite> 
```

Reply:
no, notice that we don't have the mainnet one. We have it for other chains. What I'm pointing here is that the coingecko and cc ids are the ones pre-rebranding